pillar/elastic_stack/elasticsearch/init.sls

```diff
@@ -1,13 +1,5 @@
 {% set ENVIRONMENT = salt.grains.get('environment', 'rc-apps') %}
-{% set es_hostnames = [] %}
-{% for host, addr in salt.saltutil.runner(
-    'mine.get',
-    tgt='G@roles:elasticsearch and G@environment:{}'.format(ENVIRONMENT),
-    fun='grains.item',
-    tgt_type='compound').items() %}
-{% do es_hostnames.append(host) %}
-{% endfor %}
-{% version_data = salt.cp.get_file_str('salt://elastic_stack/version_{}.sls'.format(ENVIRONMENT))|load_yaml %}
+{% set version = salt.pkg.version('elasticsearch').split('.')[0]|int %}
```

Comment:
To guard against the case where the package isn't installed you can leave off the `int` filter here, and then below update the conditional to something like:
`if version and int(version) > 6`

Reply:
Do you think that commit 9515ee7 will do it?